GameOfRunes/GameOfRunes/GameEngine/GameMetaData.swift

```diff
@@ -20,14 +20,26 @@ class GameMetaData {
     var playerMana: Int
     var score: Int
     var selectedPowerUp: PowerUpType?
+    var levelWaves: EnemySpawnUnit
+    var levelSpawnInterval: Double
```

Comment:
NIT: Use TimeInterval for time-related variables to be more clear

Reply:
Refactored. Good point